knative-operator/rockcraft.yaml

```diff
@@ -0,0 +1,62 @@
+name: knative-operator
+summary: Knative operator
+description: "Knative operator"
+version: "1.12.4"
+license: Apache-2.0
+base: ubuntu@22.04
+platforms:
+    amd64:
+run-user: _daemon_
+
+# Required due to the go codebase relying on the OS Env being set
+# See https://github.com/knative/operator/blob/knative-v1.12.4/pkg/reconciler/common/releases.go#L36
+environment:
+  KO_DATA_PATH: "/var/run/ko"
+
+services:
+  knative-operator:
+    override: replace
+    summary: "Knative operator service"
+    startup: enabled
+    command: "/ko-app/operator"
+
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
+      dpkg-query --root=${CRAFT_PROJECT_DIR}/../bundles/ubuntu-22.04/rootfs/ -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
+      > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+
+  operator:
+    plugin: go
+    source: https://github.com/knative/operator
+    source-type: git
+    source-tag: knative-v1.12.4
+    build-snaps:
+      - go/1.19/stable
+    build-environment:
+      - CGO_ENABLED: 0
+      - GOOS: linux
+    stage-packages:
```

Comment:
Inspecting the image `gcr.io/knative-releases/knative.dev/operator/cmd/operator:v1.12.4` I see that they are also including the `netbase` package (layer `908e75a94338694fe6aecf063df79fd09fe642d455886cb574832a07b0996d60`)

I'd suggest we install that one as well

Reply:
I didn't add it here because it comes with jammy base by default, but actually there's no downside to explicitly having it in stage packages

Reply:
better to have it explicit to keep consistency with all packages, committed